db/migrate/20191002103406_remove_quadicon_settings.rb

```diff
@@ -0,0 +1,18 @@
+class RemoveQuadiconSettings < ActiveRecord::Migration[5.0]
+  class User < ActiveRecord::Base
+    serialize :settings, Hash
+  end
+
+  def up
+    say_with_time("Dropping quadicon settings keys from user settings") do
+      User.all.each do |user|
```

Comment:
do we think there will be a lot of these?

If not, does it make sense to only update users with quadicon settings?

```suggestion
      User.where("settings like '%quadicons%'").each do |user|
```